### PR TITLE
Fix the text in banner block for desktop view

### PR DESCRIPTION
**Pull request by eds-kunal**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #189

Test URLs:
- Before: https://main--world-bank--aemsites.hlx.live/
- After: https://banner-bug-fix--world-bank--aemsites.hlx.live/

Here is the fix for the banner text (columns block variation). Added "banner-text" class name and added "1" as an attribute to the columns no. dropdown for the properties in columns block. This fix is for desktop view.

### CODE REVIEW

File: blocks/columns/columns.css [hunk 1]
```diff
@@ -547,5 +547,35 @@
         font-size: var(--h2-base);
         line-height: 44px;
   }
+
+  .columns.columns-1-cols.banner-text {
```
Comment (thomsebastin) [inline blocks/columns/columns.css:551]: We have decided to remove nesting due to compatibility issues in Safari. Could you please remove nesting and push changes again?

**Review by thomsebastin — CHANGES_REQUESTED** [review]

**Comment by mkbansal1** [comment]
Which design we are trying to achieve here?

File: blocks/columns/columns.css (hunk 1 above)
Comment (eds-kunal) [reply blocks/columns/columns.css:551]: Okay @thomsebastin. Will do.

**Comment by eds-kunal** [comment]
> Which design we are trying to achieve here?

banner-who-we-are-page

This is in who-we-are page.

**Pull request closed without merging**